website/docs/r/identity_oidc_provider.html.md

```diff
@@ -0,0 +1,98 @@
+---
+layout: "vault"
+page_title: "Vault: vault_identity_oidc_provider resource"
+sidebar_current: "docs-vault-identity-oidc-provider"
+description: |-
+    Provision OIDC Providers in Vault.
+---
+
+# vault\_identity\_oidc\_provider
+
+Manages OIDC Providers in a Vault server. See the [Vault documentation](https://www.vaultproject.io/api-docs/secret/identity/oidc-provider#create-or-update-an-assignment)
+for more information.
+
+## Example Usage
+
+```hcl
+resource "vault_identity_oidc_key" "test" {
+  name               = "default"
```

Comment:
Similar comment as above regarding "default" key.

Reply:
Thanks for catching these! Fixed in latest commit